rapidconnect/app/views/welcome.erb

```diff
@@ -51,32 +51,32 @@
             </div>
             <center>
               <ul class="clean">
-                <li><a href="/developers" class="btn btn-large btn-success">Get Started!</a></li>
+                <li><a href="/developers" class="btn btn-lg btn-success">Get Started!</a></li>
                 <li>&nbsp;</li>
-                <li><a href="/registration" class="btn btn-large btn-info">Register a service</a><li>
+                <li><a href="/registration" class="btn btn-lg btn-info">Register a service</a><li>
               </ul>
             </center>
           </div>
         </div>
 
         <hr>
 
-        <h3 class="muted">Example Implementations</h3>
```

Comment:
Not directly related to this change, but the example implementation don't appear to actually exist anymore. The repos do, but not the example service.

Given that we'd rather people not use RapidConnect, it might be worth just deleting the section completely.

Reply:
Good point.  I noticed it too ... but thanks for the nudge to action.  Yes, we should at the very least delete the non-functional links.

And good point about steering users towards OpenID Connect instead.

I'll sketch up a revised landing page and put it into a new PR if that's OK.

Reply:
Addressed in #20.